neo4j/README.md

```diff
@@ -2,11 +2,20 @@
 
 ## Overview
 
-This check monitors [neo4j][1] through the Datadog Agent.
-Please verify the metrics and checks that are submitted through this agent. Since Neo4j 4.0 and onward supports multiple databases, some metrics and checks are no longer published.
+The Neo4j integration for Datadog enables data collection and alerting on a [Neo4j][1] metrics available at the Prometheus endpoint, using the Datadog platform.
 
-## Setup
+The integration allows you to monitor standalone Neo4j instances as well as Neo4j causal cluster instances.
+
+## Updates
+In [Neo4j 5][9], all metric names now include dbms or database namespaces.
+
+Default dashboards for Neo4j 4 and Neo4j 5 are included in this check.
+
+This check has been tested against Neo4j 4 and Neo4j 5 databases. If there are any issues, contact DataDog support or support@neo4j.com.
```

Comment:
We don't need an update section here, instead, let's break this down to neo4j v4 and neo4j v5 and what is collected. example 

This integration supports Neo4j version 4 and Neo4j version 5. 

### Neo4j version 4
The following data is collected and provided in the out-of-the-box dashboards:
<data> 

### Neo4j version 5
The following data is collected and provided in the out-of-the-box dashboards:

Please note each version collects a different set of metrics. The versions are listed in the description of the metric.

Reply:
I think this is all done.